docs/build/guides/transactions/sponsored-reserves.mdx

```diff
@@ -180,143 +182,153 @@ func main() {
 
 Now, let’s sponsor trustlines for Account A. Notice how the `CHANGE_TRUST` operation is sandwiched between the begin and end sponsoring operations and that all relevant accounts need to sign the transaction.
 
+<Details summary="Sponsoring a single trustline">
```

Comment:
I'm using Details [here](https://developers-pr1788.previews.kube001.services.stellar-ops.com/docs/build/guides/transactions/send-and-receive-payments) as well, only difference I think is have Details on outside and CodeExample innner, and the summary prints as label in pull down in that case.

Reply:
Turned out to be a hiccup with extra `s in my code blocks!